Spigot-API-Patches/0186-Ticket-level-per-player-view-distance-API.patch

```diff
@@ -0,0 +1,84 @@
+From 798ac87ffa5eafec9179b8ff09e657174e07b11a Mon Sep 17 00:00:00 2001
+From: slicklibro <slicklibro@gmail.com>
+Date: Sat, 28 Sep 2019 00:01:23 +1200
+Subject: [PATCH] Ticket-level per player view distance API
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index 8d8d8957..ae2fe927 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -1860,18 +1860,65 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+     public void setAffectsSpawning(boolean affects);
+ 
+     /**
+-     * Gets the view distance for this player
++     * Gets the ticket view distance for this player
++     * <br>
++     * Returns the ticket view distance.
+      *
+-     * @return the player's view distance
++     * @return the player's ticket view distance
+      */
++    @Deprecated
```

Comment:
revert this, we are not deprecating this api

Reply:
see below